lib/wda-mac.js

```diff
@@ -142,6 +143,11 @@ class WDAMacProcess {
       return false;
     }
 
+    if (_.isString(opts.webDriverAgentMacUrl)) {
```

Comment:
it would always be a string because of caps validation, simply replace with `if (opts.webDriverAgentMacUrl)`

Reply:
ah... undefined can handle in `if` (i forgot it..)